### DISCUSSION TITLE
How to access the main "html" property.

### DISCUSSION THREAD

**Discussion by iYonga** [DirectoryLister/DirectoryLister#611] [Help & Support]
I want to add some custom HTML widgets to the directory page, how do I access it?

**Comment by PHLAK** [comment]
I assume you're confused about the missing `<html>` element. This is [intentional](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags) and `<html>` element is implicitly added by the browser during render. Can you elaborate on what you're trying to do and any specific issues you're having?

**Reply by iYonga** [reply]
I simply want to add stuff to the footer of the web page.

**Reply by iYonga** [reply]
Your answer in the favicon topic helped! Thanks!
